test/Targets/AIEGenerateCDODirect/01_precompiled_core_function.mlir

```diff
@@ -0,0 +1,34 @@
+// (c) Copyright 2023 Advanced Micro Devices, Inc.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// REQUIRES: cdo_direct_generation
+//
+// RUN: export BASENAME=$(basename %s)
+// RUN: rm -rf *.elf* *.xclbin *.bin $BASENAME.cdo_direct $BASENAME.prj
+// RUN: %python aiecc.py --aie-generate-cdo --no-compile-host --tmpdir $BASENAME.prj %s
+// RUN: mkdir $BASENAME.cdo_direct && cp $BASENAME.prj/*.elf $BASENAME.cdo_direct
```

Comment:
If/when we rip out aiecc.py --aie-generate-cdo, how do we test these?  Can we independently run these tests and (additionally) compare with the --aie-generate-cdo flow?

Reply:
As jeff says, we don't need both and so we won't even need these tests. This battery of tests is already like comparing a xerox-copy to an already xeros-copied copy of the canonical thing (i.e., I've found one or two bugs in the current CDO generation in this process).